assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json

```diff
@@ -4,7 +4,7 @@
         "assignedBy": "PlatOps"
       },
       "description": "Policy to check required tags",
-      "displayName": "HMCTS Tagging - CFT SBOX",
+      "displayName": "HMCTS Tagging - CFT SBOX (DCD-CFTAPPS-SBOX)",
```

Comment:
Changed the display name to keep it consistent with others, as we have more than one subscription per environment (so it can be confusing to name it 1:1)